docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -161,71 +161,36 @@ And let's build the native executable again:
- * in a browser, go to `http://localhost:8080/country/name/greece`
- * you will have an Internal Server Error
- * in your terminal, you should have an exception: `java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty`
- * hit `Ctrl+C` to stop the application
+As a consequence, system properties such as `javax.net.ssl.trustStore` do not have an effect at
+run time, so when the defaults need to be changed, these system properties must be provided at image build time.
+The easiest way to do so is by setting `quarkus.native.additional-build-args`. For example:
 
-To make it work, you need to manually set `javax.net.ssl.trustStore` to point to the new GraalVM home:
 ```
-./target/rest-client-1.0-SNAPSHOT-runner -Djavax.net.ssl.trustStore=<new-graalvm-home>/jre/lib/security/cacerts
+quarkus.native.additional-build-args=-J-Djavax.net.ssl.trustStore=/tmp/mycerts,-J-Djavax.net.ssl.trustStorePassword=changeit
 ```
 
-Now, the application should work as expected:
+will ensure that the contents of `/tmp/mycerts` are baked into the native binary and used *in addition* to the default cacerts.
 
- * in a browser, go to `http://localhost:8080/country/name/greece`
- * you should see a JSON output with some information about Greece
- * hit `Ctrl+C` to stop the application
-
-[TIP]
+[IMPORTANT]
 ====
-The root certificates file of GraalVM might not be totally up to date.
-If you have issues with some certificates, your best bet is to include the `cacerts` file of a regular JDK instead.
-====
-
-[WARNING]
-====
-Don't forget to move your GraalVM directory back to where it was.
+The file containing the custom TrustStore does *not* have to be present at runtime as its contents have been baked into the native binary.
 ====
 
 === Working with containers
 
-When working with containers, the idea is to bundle the certificates in the container and to point your binary to them using the system property mentioned above.
-
-You can for example modify your `Dockerfile.native` as follows to copy the required files to your final image:
-
-[source, subs=attributes+]
-----
-FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} as nativebuilder
-RUN mkdir -p /tmp/ssl \
-  && cp /opt/graalvm/lib/security/cacerts /tmp/ssl/
-
-FROM registry.access.redhat.com/ubi8/ubi-minimal
-WORKDIR /work/
-COPY --from=nativebuilder /tmp/ssl/ /work/
-COPY target/*-runner /work/application
-
-# set up permissions for user `1001`
-RUN chmod 775 /work /work/application \
-  && chown -R 1001 /work \
-  && chmod -R "g+rwX" /work \
-  && chown -R 1001:root /work
-
-EXPOSE 8080
-USER 1001
-
-CMD ["./application", "-Dquarkus.http.host=0.0.0.0", "-Djavax.net.ssl.trustStore=/work/cacerts"]
-----
+No special action needs to be taken when running the native binary in a container. If the native binary was properly built with the custom TrustStore
+as described in the previous section, it will properly in container as well.
```

Comment:
> it will properly in container as well

Looks like some word is missing here? Or maybe not but the meaning is not exactly obvious when you read it.

Reply:
Of course I was missing the verb :)